src/main/java/org/takes/rq/RqMultipart.java

```diff
@@ -573,13 +560,15 @@ private static StringBuilder fakeBody(final Request... parts)
                     ),
                     String.format(
                         "Content-Length: %s",
-                        RqMultipart.Fake.fakeBody(this.parts).length()
+                        this.body.length()
```

Comment:
@dalifreire these lines can be joined making the whole `String.format` one single line.